src/main/cpp/chassis/SwerveModule.cpp

```diff
@@ -98,6 +99,7 @@ SwerveModule::SwerveModule(string canbusname,
     ReadConstants(configfilename);
     InitDriveMotor(driveInverted);
     InitTurnMotorEncoder(turnInverted, canCoderInverted, angleOffset, attrs);
+    m_tractionController = std::make_unique<TractionControlController>(1.2, 1.0, 0.4, 145.0, m_maxSpeed);
```

Comment:
read from xml

Reply:
James and Aden have a branch for after kettering